README.md

```diff
@@ -38,13 +38,7 @@ For best results, dedicate a terminal to tailing `$TMPDIR/q` while you work.
 
 ```sh
 go get -u github.com/y0ssar1an/q
-```
-
-Put these aliases in your shell config. Typing `qq` will then start tailing
-`$TMPDIR/q`.
-```sh
-alias qq=". $GOPATH/src/github.com/y0ssar1an/q/q.sh"
-alias rmqq="rm $TMPDIR/q"
+cd $GOPATH/src/github.com/y0ssar1an/q && go get ./...
```

Comment:
You can avoid requiring users to `go get` the dependencies (deps) by "vendoring" them. Vendoring means copying the source code of the deps to the `vendor/` of this repo. Then, when somebody types `go get -u github.com/y0ssar1an/q`, it will fetch all the deps along with it. There's no need to recursively fetch them with `go get ./...`.

There are many vendoring tools to help you with this. Some of the most popular ones are Glide, govendor, and Godep. I use `gvt` with this repo. The command for vendoring `hpcloud/tail` would be `gvt fetch github.com/hpcloud/tail`.

Don't use Godep for vendoring. That should be considered a first-gen vendoring tool. Glide and gvt are second-gen tools that are superior because they don't pollute your `$GOPATH`. An official vendoring tool will be built in to `go` in the near future (currently scheduled for Go 1.9).